src/BlocHost-Morphic/BlMorphicSpaceHostMorph.class.st

```diff
@@ -23,7 +23,12 @@ BlMorphicSpaceHostMorph >> drawOn: aCanvas [
 		ifTrue: [ ^ self ].
 	hostSpace ifNotNil: [ 
 		aCanvas clipBy: self fullBounds during: [ :aClippedCanvas |
-			aClippedCanvas drawImage: spaceForm at: self position ] ]
+			SystemVersion current major >= 12 ifTrue: [
+				| formSet |
+				formSet := (Smalltalk at: #FormSet) extent: self extent depth: spaceForm depth forms: { spaceForm }.
```

Comment:
@tesonep @Ducasse is there a better way to access a class instead of the Smalltalk global?